test/runner/rendererStrategies/test.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA ;
+ */
+define([
+    'core/logger',
+    'taoQtiItem/runner/rendererStrategies'
+], function (loggerFactory, rendererStrategies) {
+
+    QUnit.module('rendererStrategies API');
+
+    QUnit.test('module', function (assert) {
+        assert.notEqual(typeof rendererStrategies, 'undefined', 'The module exports something');
+        assert.equal(typeof rendererStrategies, 'function', 'The module exports a function');
+    });
+
+    QUnit.cases.init([
+        { title: 'registerProvider' },
+        { title: 'getProvider' },
+        { title: 'getAvailableProviders' },
+        { title: 'clearProviders' }
+    ]).test('registry API ', function (data, assert) {
+        assert.equal(typeof rendererStrategies[data.title], 'function', `The registry exposes the method ${data.title}`);
+    });
+
+    QUnit.module('Providers');
+
+    QUnit.cases.init([
+        { title: 'commonRenderer' },
+        { title: 'reviewRenderer' }
+    ]).test('provider registered ', function (data, assert) {
+        var provider = rendererStrategies.getProvider(data.title);
```

Comment:
Yes, because of IE. All tests have been written in ES5 so far. However, it looks like this doesn't help, and tests cannot be loaded with IE actually.
I will convert this file to ES6 then.